<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>entrydialog</class>
 <widget class="QDialog" name="entrydialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Code:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Subcode:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Time Spent:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Description:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditDate">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(136, 138, 133);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxProjects"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSubcode">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(238, 238, 236);
color:rgb(0, 0, 170);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTime">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(238, 238, 236);
color:rgb(0, 0, 170);
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDescription">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(238, 238, 236);
color:rgb(0, 0, 170);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>entrydialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>entrydialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
